Content.Shared/Projectiles/SharedProjectileSystem.cs

```diff
@@ -169,32 +170,55 @@ private void OnAttemptPacifiedThrow(Entity<EmbeddableProjectileComponent> ent, r
     /// </summary>
     private void AfterProjectileHit(EntityUid uid, ProjectileComponent component, ref AfterProjectileHitEvent args)
     {
-        if (!TryComp<CanPenetrateComponent>(uid, out var damageAfterCollide))
+        if (!TryComp<CanPenetrateComponent>(uid, out var canPenetrate))
             return;
```

Comment:
You shouldn't name components with "CanX" unless it's for like a whitelist because this still has a bunch of data being handled. It'd be like calling GunComponent "CanGun" which is redundant.